templates/support/index.html

```diff
@@ -685,9 +685,46 @@ <h2 class="u-sv3">2种支持级别可供选择</h2>
     </div>
   </section>
 
+  <section class="p-strip">
+    <div class="u-fixed-width">
+      <h2 class="u-align--center">我们的合作伙伴</h2>
+        <p class="p-logo-section__title u-align--center">我们的合作伙伴为您的多样化的Ubuntu操作系统采购需求提供帮助。您也可以<a href="/contact">联系我们</a>以获得合作伙伴的联系信息。</p>
+    </div>    
+    <div class="u-fixed-width">       
+      <div class="p-logo-section__items">
+        <div class="p-logo-section__item">
+          {{
+            image(
+            url="https://assets.ubuntu.com/v1/1fb7ac1b-shenzhen-eden.png",
+            alt="Shenzhen Eden logo",
```

Comment:
Screen readers will announce images as "Image" followed by whatever string is given in the alt attribute, so there's no need to indicate these are logos in text:
```suggestion
            alt="Shenzhen Eden",
```